templates/squid.conf.header.erb

```diff
@@ -19,6 +19,14 @@ access_log                    <%= @access_log %>
 coredump_dir                  <%= @coredump_dir %>
 
 <% end -%>
+
+<% unless @error_directory.nil? -%>
```

Comment:
Just a simple ` <% if @error_directory -%>` should be fine here. The default of `undef ` will be `false`. THat looks to be how the other lines work.

Reply:
+1  `unless @foo.nil?` is only needed if the parameter is a boolean.